### PR TITLE
Update names of Boba chains

### PR DESCRIPTION
**Pull request by Boyuan-Chen**
This PR includes the following changes:
- Update the names of Boba chains (Bobabeam, Bobabase, Bobaopera, etc)
- Add Bobaopera (BOBA L2 on Fantom)
- Add Bobaavax (BOBA L2 on Avalanche)

### CODE REVIEW

**Review by tazimamatovzeka — APPROVED** [review]

**Comment by InoMurko** [comment]
hi @ligi is anything missing? thanks!

**Comment by InoMurko** [comment]
> This PR includes the following changes:
> 
> * Update the names of Boba chains (Bobabeam, Bobabase, Bobaopera, etc)
> * Add Bobaopera (BOBA L2 on Fantom)
> * Add Bobaavax (BOBA L2 on Avalanche)

Added Boba Goerli (BOBA L2 on Goerli)

**Comment by InoMurko** [comment]
hi @ligi soft ping, thanks!

**Comment by InoMurko** [comment]
ping @ligi

**Review by ligi — COMMENTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by InoMurko** [comment]
hey @ligi sorry for that. I think it's sorted now!

**Comment by InoMurko** [comment]
@ligi please review and merge if ok

File: _data/chains/eip155-2888.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Boba Network Goerli Testnet",
+  "chain": "ETH",
+  "rpc": ["https://goerli.boba.network/"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Görli Ether",
```
Comment (ligi) [inline _data/chains/eip155-2888.json:7]:
```suggestion
    "name": "Goerli Ether",
```

**Review by nutty323 — APPROVED** [review]

File: _data/chains/eip155-2888.json (hunk 1 above)
Comment (InoMurko) [reply _data/chains/eip155-2888.json]: sorted, thank you!

**Review by ligi — APPROVED** [review]

**Pull request merged**